test/fixtures/eventBridge/serverless.yml

```diff
@@ -29,3 +29,20 @@ functions:
   eventBusArn:
     handler: core.eventBusArn
     # Event filled by integration test
+  eventBusCFFunctionArn:
+    handler: core.eventBusCFFunctionArn
+    events:
+      - eventBridge:
+          useNativeCloudFormation: true
```

Comment:
I believe this needs to be updated.

Also _fixtures_ should be minimal. It's tests that should rather extend the config for desired testing (currently some fixtures are setup as rich, only because we did it that way in early phase, now we're rather extending some basic fixture in test file, and test against that)

Reply:
How minimal is minimal for a fixture and what is too rich?

Reply:
Any configuration that can be applied by extending an existing fixture, ideally if it's not put into fixture.

Still, in integration tests we usually also test by invoking functions, so then usually part of a fixture is function handler and function configuration (that's ok for integration tests)

What you may do is to reuse the `eventBridge` fixture, without updating it, for testing CF variant, by extending it with `configExt` option in integration tests (and simply applying there `provider.eventBridge.shouldUseCloudFormation: true`). Then technically exactly same event bridge events would be setup (as with custom resources run), but this time with CloudFormation resources - that'll probably be cleanest